bin/gaffer

```diff
@@ -268,6 +268,30 @@ if [[ -n $DELIGHT ]] ; then
 
 fi
 
+# Set up 3rd Party extensions
+##########################################################################
+
+for extension in `echo $GAFFER_EXTENSION_PATHS | sed 's/:/\n/g'`; do
+
+	echo Loading extension: $extension
```

Comment:
I wasn't sure about leaving this in, but I found it quite useful when writing an[ example extension](https://github.com/andrewkaufman/gafferExtensionDemo) to test against.

Reply:
[Silence is golden](http://www.linfo.org/rule_of_silence.html) - let's leave it out.